src/dissect/dissect.c

```diff
@@ -39,7 +41,8 @@ static void help(void) {
                "  -r --read-only       Mount read-only\n"
                "     --fsck=BOOL       Run fsck before mounting\n"
                "     --discard=MODE    Choose 'discard' mode (disabled, loop, all, crypto)\n"
-               "     --root-hash=HASH  Specify root hash for verity\n",
+               "     --root-hash=HASH  Specify root hash for verity\n"
+               "     --hash-device=PATH   Specify hash device for verity\n",
```

Comment:
please rebreak the whole table, so that things are nicely aligned again

Reply:
sure, done